docs/products/mysql/get-started.md

```diff
@@ -3,36 +3,380 @@ title: Get started with Aiven for MySQL®
+
+## Prerequisites
+
+- Access to the [Aiven Console](https://console.aiven.io)
+- [MySQL CLI client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html)
+  installed
+- [Terraform installed](https://developer.hashicorp.com/terraform/install) if you prefer
+  to get started using code
 
 ## Create a service
 
-[Create an Aiven for PostgreSQL® service](/docs/platform/howto/create_new_service) in the
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+[Create an Aiven for MySQL® service](/docs/platform/howto/create_new_service)  in the
 [Aiven Console](https://console.aiven.io).
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+1. [Create an authentication token](/docs/platform/howto/create_authentication_token).
+1. Store the authentication token in an environment variable:
+
+   ```bash
+   export TF_VAR_aiven_api_token=YOUR_AIVEN_API_TOKEN
+   ```
+
+1. Create the following Terraform files:
+
+   - ``provider.tf``, where you specify the version in the ``required_providers`` block
+
+      ```hcl
+      terraform {
+        required_providers {
+          aiven = {
+            source  = "aiven/aiven"
+            version = ">=4.0.0, < 5.0.0"
+          }
+        }
+      }
+
+      provider "aiven" {
+        api_token = var.aiven_api_token
+      }
+      ```
+
+   - ``mysql.tf``, where you include the ``aiven_mysql`` resource
+
+      ```hcl
+      resource "aiven_mysql" "mysql" {
+        project      = var.project_name
+        service_name = var.service_name
+        cloud_name   = var.cloud_name
+        plan         = var.service_plan
+      }
+
+      output "mysql_service_host" {
+        value = aiven_mysql.mysql.service_host
+      }
+
+      output "mysql_service_port" {
+        value = aiven_mysql.mysql.service_port
+      }
+
+      output "mysql_service_username" {
+        value = aiven_mysql.mysql.service_username
+      }
+
+      output "mysql_service_password" {
+        value     = aiven_mysql.mysql.service_password
+        sensitive = true
+      }
+      ```
+
+   - ``variables.tf``, where you declare the API token and project name variables
 
-The new service appears on the **Services** page and its status
-indicator shows **Rebuilding**. It usually takes a couple of minutes before the status
-changes to **Running**.
+      ```hcl
+      variable "aiven_api_token" {
+        description = "Aiven API token"
+        type        = string
+      }
+
+      variable "project_name" {
+        description = "Project name"
+        type        = string
+      }
+
+      variable "cloud_name" {
+        description = "Cloud name"
+        type        = string
+      }
+
+      variable "service_name" {
+        description = "Service name"
+        type        = string
+      }
+
+      variable "service_plan" {
+        description = "Service plan"
+        type        = string
+      }
+      ```
+
+   - ``terraform.tfvars``, where you add the Aiven access token and project name
```

Comment:
We do not assign a value for the API token here, so same here as well, I would just write, e.g. ", where you assign actual values to your previously declared variables."